README.md

```diff
@@ -16,6 +16,7 @@ The following table shows the relation between the versions of the two projects:
 | 1.7.x | 0.0.4 |
 | 1.8.x | 0.0.6 |
 | 1.9.x | 0.0.7 |
+| 1.10.x | 0.1.0 |
```

Comment:
Should we consider inverting the order of this table? (Latest release first)

Reply:
That makes sense to me.